packages/turf-rbush/LICENSE.geojson-rbush

```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
```

Comment:
Is this license block the same as `LICENSE` with the exception of crediting Denis Carriere? We may need to just figure out how to combine the two and get rid of this one.

Reply:
Just being extra cautious to keep as hadn't heard from Denis, and wasn't 100% of the history. Will see if merging is possible.

Reply:
I think the package.json has a contributors field, but I'm not a lawyer so I don't fully understand the nuance of having him listed directly in the license file vs being there. I'm happy enough to just change the testing to omit this license file from the checks if we have to.